File: MascotasUN/src/main/java/uiMain/gestion/gestionUsuarios/EliminarUsuario.java
package uiMain.gestion.gestionUsuarios;

import gestorAplicacion.gestionPersonas.Usuario;
import gestorAplicacion.gestionVentas.Producto;
import gestorAplicacion.gestionVentas.Tienda;

import java.util.Scanner;

public class EliminarUsuario {
    private static int id;
    public static void eliminarUsuario(){
        Scanner input = new Scanner(System.in);

        while(true){
            System.out.println("Ingrese el codigo del usuario que se quiere eliminar: ");
            id=input.nextInt();

            //Haciendo uso del metodo de eliminar una instancia de la clase usuario, se elimina el objeto con el id indicado
            boolean usuarioELiminado = Usuario.eliminarUsuario(Usuario.encontrarPersona(id));
            if (usuarioELiminado ==false){
                System.out.println("No se encontro el usuario, intente nuevamente");
            } else {
                System.out.println("Usuario Eliminado");
                //System.out.println(Usuario.getUsuarios()); Si quieren ver el uso de eliminar
                break;
            }
        }
    }
}

File: Practica2-MascotasUN/src/uiMain/gestion/gestionUsuarios/EliminarUsuario.py
class EliminarUsuario:
    
    @classmethod
    def eliminarUsuario(cls):
        print("Estás a punto de eliminar un usuario del sistema")
        id = input("Ingrese el codigo del usuario que se quiere eliminar: ")
        if id in Tienda.usuarios:
            del Tienda.usuarios[id]
            print("Usuario eliminado exitosamente")
        else:
            print("El usuario no existe")